### PR TITLE
Switched back to official segment snippet.

### PR DESCRIPTION
**Pull request by jordanh**
Maintainers fixed for webpack. See: https://github.com/segmentio/snippet/issues/21#issuecomment-247172532

### CODE REVIEW

File: src/server/Html.js [hunk 1]
```diff
@@ -45,7 +43,17 @@ export default function Html({
         />
         <title>{title}</title>
         {/* segment.io analytics */}
-        {process.env.SEGMENT_WRITE_KEY && <script type="text/javascript" async src={segmentCdnSrc}/>}
+        {process.env.SEGMENT_WRITE_KEY &&
+          <script
+            type="text/javascript"
+            dangerouslySetInnerHTML={{
+              __html: segmentSnippet.min({
```
Comment (mattkrick) [inline src/server/Html.js]: Should we do this outside the component?

Comment (jordanh) [inline src/server/Html.js]: You mean, generate the snippet? Yes, probably. One moment...

File: src/server/Html.js [hunk 2]
```diff
@@ -46,12 +51,7 @@ export default function Html({
         {process.env.SEGMENT_WRITE_KEY &&
           <script
             type="text/javascript"
-            dangerouslySetInnerHTML={{
-              __html: segmentSnippet.min({
-                host: 'cdn.segment.com',
-                apiKey: process.env.SEGMENT_WRITE_KEY
-              })
-            }}
+            dangerouslySetInnerHTML={{__html: segmentSnippet}}
```
Comment (jordanh) [inline src/server/Html.js]: We good now @mattkrick ?

Comment (mattkrick) [inline src/server/Html.js]: LGTM 👍

**Pull request merged**